test/main.js

```diff
@@ -601,8 +601,9 @@ describe('node-fetch', () => {
 		return fetch(url).then(res => {
 			expect(res.status).to.equal(200);
 			expect(res.ok).to.be.true;
-			return expect(res.text()).to.eventually.be.rejectedWith(Error)
-				.and.have.property('message').includes('Premature close');
```

Comment:
Hmmm, this should still reject with an error, right? Seems like the new code is missing to check for errors at some point? 🤔

Reply:
It's Draft PR, not ready for PR yet.
But yes, I'm struggling to understand expected behavior with a partial body. Do we need to put a partial body somewhere or just discard it?

Reply:
Hehe, sorry, I was a bit eager 😁 

I think that we can just discard it, since the request have errored 🤔